src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -169,9 +165,7 @@ public function set($id, $service)
             throw new InvalidArgumentException('You cannot set service "service_container".');
         }
 
-        if (isset($this->aliases[$id])) {
-            @trigger_error(sprintf('Replacing the "%s" (alias of "%s") service is deprecated since Symfony 3.2 and won\'t be supported anymore in Symfony 4.0.', $id, $this->aliases[$id]), E_USER_DEPRECATED);
-        } elseif (isset($this->services[$id])) {
+        if ($this->has($id)) {
             @trigger_error(sprintf('Replacing the "%s" service is deprecated since Symfony 3.2 and won\'t be supported anymore in Symfony 4.0.', $id), E_USER_DEPRECATED);
```

Comment:
```diff
- Replacing the "%s" service
+ Replacing the "%s" service or alias 
```

?

Why not keeping the previous `Replacing the "%s" (alias of "%s") service` check & trigger + the `elseif` to be more accurate on the message ?